Makefile

```diff
@@ -39,10 +43,45 @@ lint: dependencies-frontend
 	$(YARN) lint
 
 assets: build dependencies-frontend
-	$(BINDATA) -pkg asset -o ./server/asset/asset.go `find ./build -type d`
+	chmod -R go=r $(ASSETS_PATH); \
+	$(BINDATA) \
+		-modtime 1 \
+		-pkg asset \
+		-o ./server/asset/asset.go \
+		-prefix $(BASE_PATH) \
+		$(ASSETS_PATH)/...
 
 build: dependencies-frontend
-	REACT_APP_SERVER_URL=$(SERVER_URL) $(YARN) build
+	GENERATE_SOURCEMAP=false REACT_APP_SERVER_URL=$(SERVER_URL) $(YARN) build
+
+validate-commit: assets
+	git status --untracked-files=no --porcelain | grep -qe '..*'; \
+	if  [ $$? -eq 0 ] ; then \
+		git diff|cat; \
+		echo >&2 "generated bindata is out of sync"; \
+		exit 2; \
+	fi
+
+# copy-past from CI but without CGO_ENABLED=0
+packages:
```

Comment:
This is (almost) already implemented in [src-d/ci/Makefile.main#L117](https://github.com/src-d/ci/blob/master/Makefile.main#L117), why not fixing or improving its problems there, and then reuse it here?

Reply:
I'm sure there is a reason for `CGO_ENABLED=0`, other projects don't use client go.

Reply:
I think that if we do `CGO_ENABLED=1` it'd work for dashboard; so then we could pass to `src-d/ci/Makefile.main::packages` a `1` value for `CGO_ENABLED`, keeping the `0` value as the default one.

Doing so, `CGO_ENABLED` could be `0` as default, or `1` for projects needing it.

Reply:
I don't understand you. Do you suggest add default variable CGO_ENABLED in CI repo and override it here? If yes, I think it's a good solution. I'll do PR to CI. But then we have broken CI until it got merged in CI. I prefer to fix our project first and then improve company-wide it. (same as above)

Reply:
Yes, you understood it perfectly! :)
Could you please create the `src-d/ci `PR? Once ir ir ready we'll be able to start pushing for it to be merged 😉